requirements.txt

```diff
@@ -21,9 +21,11 @@ joblib>=1.1.0
 jupyter-server-proxy==3.2.1
 jupyter-server==1.23.5
 jupyter_client==7.3.4
+langchain>=0.0.222
```

Comment:
Let's make add these to the extra packages.
* Move it below the `# extras` comment
* Add a section here (e.g. ai) https://github.com/mage-ai/mage-ai/blob/master/setup.py#L42
* Add the two packages to the ai section and all section

Reply:
Updated